app/Services/GoogleSheetsService.php

```
@@ -112,6 +114,10 @@ public function syncFormalization(string $spreadsheetId, string $sheetName, int
 
             try {
                 Formalization::updateOrCreate(['project_id' => $project->id], $record);
+                $nup = Import::normalizeNup($row[$config['opening_nup_column'] ?? 'N° DO PROCESSO (NUP)'] ?? null);
+                if ($nup !== null) {
+                    Opening::where('project_id', $project->id)->update(['opening_nup' => $nup]);
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Opening::where().


(staticMethod.notFound)